src/dbnode/client/options.go

```diff
@@ -199,6 +199,12 @@ var (
 			SetJitter(true),
 	)
 
+	// defaultChannelOptions are default tchannel channel options.
+	defaultChannelOptions = &tchannel.ChannelOptions{
+		MaxIdleTime:       5 * time.Minute,
```

Comment:
nit: Probably would've opted for 1min max idle time and 1-2 minute idle check interval, just since 1min in this world is a very long time heh.